vro-types/o11n-plugin-aria/index.d.ts

```diff
@@ -10,16 +10,429 @@ declare class VraHost {
+	 * @param cloudAccountVsphere
+	 */
+	deleteVSphereCloudAccount(
+		cloudAccountVsphere: VraCloudAccountVsphere
+	): void;
+
+	/**
+	 * Create a vSphere cloud account
+	 * @param body
+	 */
+	createVSphereCloudAccount(
+		body: VraCloudAccountVsphereSpecification
+	): VraCloudAccountVsphere;
+
+	/**
+	 * Create a vSphere cloud account in an asynchronous manner
+	 * @param body
+	 */
+	createVSphereCloudAccountAsync(
+		body: VraCloudAccountVsphereSpecification
+	): VraRequestTracker;
+
+	/**
+	 * Update vSphere cloud account in an asynchronous manner
+	 * @param cloudAccountVsphere
+	 * @param body
+	 */
+	updateVSphereCloudAccountAsync(
+		cloudAccountVsphere: VraCloudAccountVsphere,
+		body: VraUpdateCloudAccountVsphereSpecification
+	): VraRequestTracker;
+
+	/**
+	 * Update vSphere cloud account
+	 * @param cloudAccountVsphere
+	 * @param body
+	 */
+	updateVSphereCloudAccount(
+		cloudAccountVsphere: VraCloudAccountVsphere,
+		body: VraUpdateCloudAccountVsphereSpecification
+	): VraCloudAccountVsphere;
+}
+
+declare class VraUpdateCloudAccountVsphereSpecification {
+	hostName: string;
+	acceptSelfSignedCertificate: boolean;
+	createDefaultZones: boolean;
+	password: string;
+	associatedCloudAccountIds: Object[];
+	regions: Object[];
+	dcid: string;
+	name: string;
+	description: string;
+	username: string;
+	tags: Object[];
+	constructor();
+	constructor();
+
+	/**
+	 * @param associatedCloudAccountIdsItem
+	 */
+	addAssociatedCloudAccountIdsItem(
+		associatedCloudAccountIdsItem: string
+	): VraUpdateCloudAccountVsphereSpecification;
+
+	/**
+	 * @param tagsItem
+	 */
+	addTagsItem(tagsItem: VraTag): VraUpdateCloudAccountVsphereSpecification;
+
+	/**
+	 * @param regionsItem
+	 */
+	addRegionsItem(
+		regionsItem: VraRegionSpecification
+	): VraUpdateCloudAccountVsphereSpecification;
+}
+
+/**
+ * Specification for a vSphere cloud account.<br><br>A cloud account identifies a cloud account type and an account-specific deployment region or data center where the associated cloud account resources are hosted.
+ */
+declare class VraCloudAccountVsphereSpecification {
+	hostName: string;
+	acceptSelfSignedCertificate: boolean;
+	createDefaultZones: boolean;
+	password: string;
+	associatedCloudAccountIds: Object[];
+	regions: Object[];
+	dcid: string;
+	name: string;
+	description: string;
+	username: string;
+	tags: Object[];
+	constructor();
+	constructor();
+
+	/**
+	 * @param regionsItem
+	 */
+	addRegionsItem(
+		regionsItem: VraRegionSpecification
+	): VraCloudAccountVsphereSpecification;
+
+	/**
+	 * @param associatedCloudAccountIdsItem
+	 */
+	addAssociatedCloudAccountIdsItem(
+		associatedCloudAccountIdsItem: string
+	): VraCloudAccountVsphereSpecification;
+
+	/**
+	 * @param tagsItem
+	 */
+	addTagsItem(tagsItem: VraTag): VraCloudAccountVsphereSpecification;
+}
+
+declare class VraRegionSpecification {
+	externalRegionId: string;
+	name: string;
+	constructor();
```

Comment:
Duplicate

Reply:
Duplicate what?

Reply:
I don't see that. The same for lines 223,224

![image](https://github.com/user-attachments/assets/3270737b-ccc0-4d70-b3a0-ee65f457dd63)

Reply:
Did you push your latest changes?

Reply:
not yet

Reply:
But I did remove all constructors duplicates everywhere in general.

Reply:
Ok, I will verify that as soon as the changes have been pushed.